### ISSUE TITLE
AsyncAPI

### ISSUE THREAD

**Issue by timburks** [google/gnostic#243]
The [Async API]https://www.asyncapi.com/) spec is very similar in structure to OpenAPI. With a JSON schema, we might generate a .proto model and compiler for it.

**Comment by LorenzHW** [comment]
Yes, I am in! I can contribute with gnostic-grpc-async. I think you mentioned it over [here](https://github.com/google/gnostic/issues/212#issuecomment-695328346) as well Tim.

**Comment by morphar** [comment]
I am very interested in this as well! I might not be able to put to much time in it at the moment, but in the not to distant future, I should be able to find some time for it, if that would help.
